styles/styles.css

```
@@ -0,0 +1,316 @@
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("fonts/ptsans-400.woff2") format("woff2");
```

Шрифты подключаются так же как и картинки, т.е. относительно CSS файла. Т.е. нужно выйти на папку выше (`..`)
```suggestion
    src: url("fonts/ptsans-400.woff2") format("woff2");
```